ios/Exponent/Kernel/Views/EXErrorView.m

```diff
@@ -94,9 +94,13 @@ - (void)setType:(EXFatalErrorType)type
       } else if (_appRecord.appLoader.manifestUrl) {
         NSString *url = _appRecord.appLoader.manifestUrl.absoluteString;
         if ([self _urlLooksLikeLAN:url]) {
+          NSString *extraLANPermissionText = @"";
+          if (@available(iOS 14, *)) {
+            extraLANPermissionText = @", and that you have granted Expo the Local Network permission in the Settings app,";
```

Comment:
we might want to commit this to master as "Expo Go" and to SDK 40 as just "Expo"

Reply:
For now I think just "Expo" is better since we'll need to cherry-pick a change to sdk-40 to change Expo to Expo Go when we release Expo Go from that branch. That is, it's easier if the Expo Go renaming happens all in its own commits.